pkgs/development/python-modules/aioextensions/default.nix

```
@@ -0,0 +1,35 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, pythonOlder
+
+# Python dependencies
+, uvloop
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "aioextensions";
+  version = "20.9.2315218";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0wfcdgk6hq52b3rpg24z9w861m8xplna3vjrrg2xg2mdwcgfsr3j";
+  };
+
+  propagatedBuildInputs = [
+    uvloop
+  ];
+
+  checkInputs = [
+    pytest
+  ];
+
```

```suggestion
  checkInputs = [
    pytest
  ];

```

running build_ext

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK

---

now it should be ready

```
nixpkgs$ nix-shell -p python38Packages.aioextensions -I nixpkgs=.
these derivations will be built:
  /nix/store/fk6xk55bl4g0rl1qbncw37mf1y1whw6a-python3.8-aioextensions-20.11.1517005.drv
building '/nix/store/fk6xk55bl4g0rl1qbncw37mf1y1whw6a-python3.8-aioextensions-20.11.1517005.drv'...
Sourcing python-recompile-bytecode-hook.sh
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing setuptools-build-hook
Using setuptoolsBuildPhase
Using setuptoolsShellHook
Sourcing pip-install-hook
Using pipInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing setuptools-check-hook
unpacking sources
unpacking source archive /nix/store/z1664w21d9vb4gx29y5z4g2chqk1ljcg-aioextensions-20.11.1517005.tar.gz
source root is aioextensions-20.11.1517005
setting SOURCE_DATE_EPOCH to timestamp 1605705810 of file aioextensions-20.11.1517005/PKG-INFO
patching sources
configuring
no configure script, doing nothing
building
Executing setuptoolsBuildPhase
running bdist_wheel
running build
running build_py
creating build
creating build/lib
creating build/lib/aioextensions
copying src/aioextensions/__init__.py -> build/lib/aioextensions
copying src/aioextensions/py.typed -> build/lib/aioextensions
installing to build/bdist.linux-x86_64/wheel
running install
running install_lib
creating build/bdist.linux-x86_64
creating build/bdist.linux-x86_64/wheel
creating build/bdist.linux-x86_64/wheel/aioextensions
copying build/lib/aioextensions/__init__.py -> build/bdist.linux-x86_64/wheel/aioextensions
copying build/lib/aioextensions/py.typed -> build/bdist.linux-x86_64/wheel/aioextensions
running install_egg_info
running egg_info
creating src/aioextensions.egg-info
writing src/aioextensions.egg-info/PKG-INFO
writing dependency_links to src/aioextensions.egg-info/dependency_links.txt
writing requirements to src/aioextensions.egg-info/requires.txt
writing top-level names to src/aioextensions.egg-info/top_level.txt
writing manifest file 'src/aioextensions.egg-info/SOURCES.txt'
reading manifest file 'src/aioextensions.egg-info/SOURCES.txt'
writing manifest file 'src/aioextensions.egg-info/SOURCES.txt'
Copying src/aioextensions.egg-info to build/bdist.linux-x86_64/wheel/aioextensions-20.11.1517005-py3.8.egg-info
running install_scripts
adding license file "LICENSE.md" (matched pattern "LICEN[CS]E*")
creating build/bdist.linux-x86_64/wheel/aioextensions-20.11.1517005.dist-info/WHEEL
creating 'dist/aioextensions-20.11.1517005-py3-none-any.whl' and adding 'build/bdist.linux-x86_64/wheel' to it
adding 'aioextensions/__init__.py'
adding 'aioextensions/py.typed'
adding 'aioextensions-20.11.1517005.dist-info/LICENSE.md'
adding 'aioextensions-20.11.1517005.dist-info/METADATA'
adding 'aioextensions-20.11.1517005.dist-info/WHEEL'
adding 'aioextensions-20.11.1517005.dist-info/top_level.txt'
adding 'aioextensions-20.11.1517005.dist-info/RECORD'
removing build/bdist.linux-x86_64/wheel
Finished executing setuptoolsBuildPhase
installing
Executing pipInstallPhase
/build/aioextensions-20.11.1517005/dist /build/aioextensions-20.11.1517005
DEPRECATION: The -b/--build/--build-dir/--build-directory option is deprecated. pip 20.3 will remove support for this functionality. A possible replacement is use the TMPDIR/TEMP/TMP environment variable, possibly combined with --no-clean. You can find discussion regarding this at https://github.com/pypa/pip/issues/8333.
Processing ./aioextensions-20.11.1517005-py3-none-any.whl
Requirement already satisfied: uvloop in /nix/store/pd3cvs7hfpw21wzi8wy9ri1m45b1figa-python3.8-uvloop-0.14.0/lib/python3.8/site-packages (from aioextensions==20.11.1517005) (0.14.0)
Installing collected packages: aioextensions
Successfully installed aioextensions-20.11.1517005
/build/aioextensions-20.11.1517005
Finished executing pipInstallPhase
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/2yrqqhwv19hcv3xyp8knhs9fl8x5sms1-python3.8-aioextensions-20.11.1517005
strip is /nix/store/79rshjqnibwcx468m4i31wnmasaj6nf4-binutils-2.31.1/bin/strip
stripping (with command strip and flags -S) in /nix/store/2yrqqhwv19hcv3xyp8knhs9fl8x5sms1-python3.8-aioextensions-20.11.1517005/lib 
patching script interpreter paths in /nix/store/2yrqqhwv19hcv3xyp8knhs9fl8x5sms1-python3.8-aioextensions-20.11.1517005
checking for references to /build/ in /nix/store/2yrqqhwv19hcv3xyp8knhs9fl8x5sms1-python3.8-aioextensions-20.11.1517005...
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
running install tests
============================= test session starts ==============================
platform linux -- Python 3.8.6, pytest-5.4.3, py-1.9.0, pluggy-0.13.1
rootdir: /build/aioextensions-20.11.1517005
collected 5 items                                                              

test___init__.py .....                                                   [100%]

============================== 5 passed in 5.85s ===============================
pythonCatchConflictsPhase
pythonRemoveBinBytecodePhase
pythonImportsCheckPhase
Executing pythonImportsCheckPhase
pytestcachePhase

[nix-shell:~/Documents/kamadorueda/nixpkgs]$ exit
```